test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -356,67 +335,57 @@ Describe "Copy-Item can avoid copying an item onto itself" -Tags "CI", "RequireA
                     Source = $symToOtherFile
                     Destination = $otherFile
-                    SkipIf = -Not $canSymlink
                 }
                 @{
                     Name = "Copy symbolic link to symbolic link with same target"
                     Source = $secondSymToOther
                     Destination = $symToOther
-                    SkipIf = -Not $canSymlink
                 }
                 @{
                     Name = "Copy through chain of symbolic links"
                     Source = $symToSym
                     Destination = $otherSubDir
-                    SkipIf = -Not $canSymlink
-                }
-
-                # Junctions and directory symbolic links are Windows and NTFS only
-                @{
-                    Name = "Copy junction to target"
-                    Source = $junctionToOther
-                    Destination = $otherSubDir
-                    SkipIf = -Not $canJunction
-                }
-                @{
-                    Name = "Copy directory symbolic link to target"
-                    Source = $symdToOther
-                    Destination = $otherSubDir
-                    SkipIf = -Not $canDirSymLink
                 }
             )
+
+            # Junctions and directory symbolic links are Windows and NTFS only
+            if ($IsWindows)
+            {
+                $testCases += @(
+                    @{
+                        Name = "Copy junction to target"
+                        Source = $junctionToOther
+                        Destination = $otherSubDir
+                    }
+                    @{
+                        Name = "Copy directory symbolic link to target"
+                        Source = $symdToOther
+                        Destination = $otherSubDir
+                    }
+                )
+            }
         }
 
-        #It "<Name>" -TestCases $testCases -Skip:($testCase.SkipIf) {
         It "<Name>" -TestCases $testCases {
             Param (
                 [string]$Name,
                 [string]$Source,
-                [string]$Destination,
-                [bool]$SelfCopyExpected
+                [string]$Destination
             )
 
-            # The source path must exist.
-            # If it does not, it's because it could not be created in the BeforeAll block
-            if (-Not (Test-Path -Path $Source))
-            {
-                return
-            }
-
-            # The destination path must exist.
-            # If it does not, it's because it could not be created in the BeforeAll block
-            if ($SelfCopyExpected -And (TestPath -Path $Destination))
+            # The source and destination paths must exist.
+            # If either do not, it's because they could not be created in the BeforeAll block
+            # This is not a test failure, but rather a precondition failure.
+            if ((-Not (Test-Path -Path $Source)) -or (-Not (Test-Path -Path $Destination)))
```

Comment:
Please clarify 'ShouldBeErrorId' below don't fail if the items don't exist?
If failed please remove the "if-return".
If not failed we should replace the `if-return` with `Should`.